scripts/run-riscv

```diff
@@ -119,7 +119,7 @@ chardev=stdio,mux=on,id=stdio0
 
 # Point the semihosting driver at our new chardev
 
-cmdline="program-name"
+cmdline="$elf" 
```

Comment:
This looks useful; the only possible snag is that the semihosting API provides the command line as a single string, and the semihosting code in picolibc splits that apart into white-space separated words. If there are spaces (or tabs) in $elf, that will cause trouble. Maybe something like
```
cmdline=$(echo "$elf" | tr ' ' '_')
```
to replace blanks with underscore?